src/rosdiscover/recover/symbolic.py

```diff
@@ -336,6 +336,9 @@ def to_dict(self) -> t.Dict[str, t.Any]:
     def eval(self, context: SymbolicContext) -> None:
         ...
 
+    def contains(self, stmt: SymbolicStatement, program_function_map: t.Mapping[str, SymbolicFunction]) -> bool:
```

Comment:
suggest renaming `program_function_map` to `name_to_function` so that the key is clear without needing to read the docs 👍🏻